File: app/src/main/java/space/lopatkin/spb/helpboardgamecard/data/repository/SettingsRepository.java
package space.lopatkin.spb.helpboardgamecard.data.repository;

public interface SettingsRepository {

    /**
     * Сохранить вариант включенной клавиатуры: кастомная или дефолтная.
     */
    void saveKeyboardVariant(int keyboardVariant);

    /**
     * Получить вариант включенной клавиатуры: кастомная или дефолтная.
     */
    int getKeyboardVariant();

}

File: app/src/main/java/space/lopatkin/spb/helpboardgamecard/data/repository/SettingsRepository.kt
package space.lopatkin.spb.helpboardgamecard.data.repository

interface SettingsRepository {
    /**
     * Сохранить тип включенной клавиатуры в настройках: кастомная или дефолтная.
     *
     * @param type выбранный пользователем тип.
     */
    fun saveKeyboardType(type: Int)

    /**
     * Получить тип включенной клавиатуры в настройках: кастомная или дефолтная.
     */
    fun getKeyboardType(): Int

}
